spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.md

```diff
@@ -213,4 +205,12 @@ Chip(
     text = "Chip",
     leadingIcon = if (selected) SparkIcon.Toggles.Check.Simple else null
 )
+```
+
+To correctly handle selectable state, pass the following modifier to components 
+ ```kotlin
+fun Modifier.semantics {
```

Comment:
I think the `selectable` or the `togglable` modifer provide this already, did you looked into it? If not we might want to verify this

Reply:
Yes, there is a `clickable` part in it + `semantics`. But we have already onClick param, so basically only `semantics` would be needed fo assistance services. 
II also had a look at `Modifier.selectable` which is the seame  `clickable` + `semantics`